pkgs/development/libraries/libshumate/default.nix

```diff
@@ -68,11 +68,14 @@ stdenv.mkDerivation rec {
 
   postFixup = ''
     # Cannot be in postInstall, otherwise _multioutDocs hook in preFixup will move right back.
-    moveToOutput share/doc/libshumate-0.0 "$devdoc"
+    moveToOutput share/doc/libshumate-${lib.versions.majorMinor version} "$devdoc"
```

Comment:
ABI version is not coupled to package version:

https://gitlab.gnome.org/GNOME/libshumate/-/blob/6404253a6298e1271e564f9fd8e521a2fd30e0f5/meson.build#L27